src/pages/account/settings/_mock.ts

```diff
@@ -0,0 +1,67 @@
+import { Request, Response } from 'express';
+import city from './geographic/city.json';
+import province from './geographic/province.json';
+
+function getProvince(_: Request, res: Response) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'getProvince' to have a typedef](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)